Week8/hardwoong/Mission2/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,73 @@
+import {useEffect, useState} from "react";
+import LpCard from "../components/LpCard/LpCard.tsx";
+import useGetInfiniteLpList from "../hooks/queries/useGetInfiniteLpList.ts";
+import {PAGINATION_ORDER} from "../enums/common.ts";
+import { useInView } from "react-intersection-observer";
+import LpCardSkeletonList from "../components/LpCard/LpCardSkeletonList.tsx";
+import SortOrderButtons from "../components/SortOrderButtons.tsx";
+import useDebounce from "../hooks/queries/useDebounce.ts";
+import { SEARCH_DEBOUNCE_DELAY } from "../constants/delay.ts";
+import useThrottle from "../hooks/queries/useThrottle";
+
+const HomePage = () => {
+  const [sortOrder, setSortOrder] = useState<"asc" | "desc">("desc");
```

Comment:
현재 `HomePage`에서 하드코딩된 "asc" | "desc" 타입을 사용하고 있고, 아래에서 `sortOrder as PAGINATION_ORDER`로 타입 단언을 사용하고 있습니다. `PAGINATION_ORDER` enum을 별도 파일에 정의해놨음에도 불구하고 문자열 리터럴 타입을 직접 사용하면 타입 정의가 중복되고 유지보수성도 떨어집니다.

정의해둔 enum을 사용해서 `useState<PAGINATION_ORDER>(PAGINATION_ORDER.DESC)` 같은 형태로 변경하고 타입 단언도 제거하면, 타입 안정성을 높이고 코드 유지보수성도 개선할 수 있을 것 같습니다!


```suggestion
  const [sortOrder, setSortOrder] = useState<PAGINATION_ORDER>(PAGINATION_ORDER.desc);
```